mockld/sdk_data.go

```diff
@@ -0,0 +1,147 @@
+package mockld
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/launchdarkly/sdk-test-harness/framework/helpers"
+
+	"gopkg.in/launchdarkly/go-jsonstream.v1/jreader"
+	"gopkg.in/launchdarkly/go-server-sdk-evaluation.v1/ldmodel"
+)
+
+type SDKKind string
+
+const (
+	ServerSideSDK SDKKind = "server"
+	MobileSDK     SDKKind = "mobile"
+	JSClientSDK   SDKKind = "jsclient"
+)
+
+type DataItemKind string
+
+type SDKData interface {
+	SDKKind() SDKKind
+	Serialize() []byte
```

Comment:
Will this serialization always be JSON?

Reply:
Yes. That is how all of the LD service endpoints work, as documented in the specs for the SDK data model and the streaming and polling services.